src/node/server/serverPluginHmr.ts

```diff
@@ -150,47 +150,43 @@ export const hmrPlugin: ServerPlugin = ({
     }
 
     const publicPath = resolver.fileToRequest(filePath)
-    const importers = importerMap.get(publicPath)
-    if (importers) {
-      const hmrBoundaries = new Set<string>()
-      const dirtyFiles = new Set<string>()
-      dirtyFiles.add(publicPath)
-
-      const hasDeadEnd = walkImportChain(
-        publicPath,
-        importers,
-        hmrBoundaries,
-        dirtyFiles
-      )
-
-      // record dirty files - this is used when HMR requests coming in with
-      // timestamp to determine what files need to be force re-fetched
-      hmrDirtyFilesMap.set(String(timestamp), dirtyFiles)
+    const importers = ensureMapEntry(importerMap, publicPath)
```

Comment:
github diff is confused. I only change this line and delete the following if branch

Reply:
Below the `Files changed` tab, you'll see `Jump to...`. Next to is a cog. Click it. Make sure ` Hide whitespace changes` is checked and click `Apply and reload`.